cicd/app/rasd_static.yaml

```diff
@@ -95,7 +95,7 @@ Resources:
       DistributionConfig:
         Aliases:
           - !Sub "${pSubDomain}.${pHostedZone}"
-          - !Ref pHostedZone
+        #  - !Ref pHostedZone
```

Comment:
temporarily commented out the creation of the root domain and it's being used in testing for the service.